src/aboutwindow.cpp

```diff
@@ -9,7 +9,7 @@
 AboutWindow::AboutWindow(QWidget *parent) : QWidget(parent), m_ui(new Ui::AboutWindow)
 {
     m_ui->setupUi(this);
-    this->setWindowFlags(Qt::Window);
+    this->setWindowFlags(Qt::SplashScreen | Qt::WindowStaysOnTopHint);
```

Comment:
It seems to me that the `AboutWindow` should really derive from `QDialog` instead. I don't see the point of `Qt::WindowStaysOnTopHint` and `Qt::SplashScreen` (which is just the same as `ToolTip | Dialog`) seems a little out of place (on Xfce, this creates a window without frame that can't be closed).

Btw, while you're changing this anyway, please remove the superfluous `this->`. :-)